Detectors/MUON/MCH/Calibration/src/channel-calib-workflow.cxx

```diff
@@ -10,14 +10,23 @@
 
 #include "MCHChannelCalibratorSpec.h"
 #include "Framework/DataProcessorSpec.h"
+#include "Framework/ConfigParamSpec.h"
+#include "Framework/CompletionPolicyHelpers.h"
 
 using namespace o2::framework;
 
+// customize the completion policy
+void customize(std::vector<o2::framework::CompletionPolicy>& policies)
+{
+  using o2::framework::CompletionPolicy;
+  policies.push_back(CompletionPolicyHelpers::defineByName("calib-mchchannel-calibration.*", CompletionPolicy::CompletionOp::Consume));
```

Comment:
The name used here must be in sync with what's used in the calibrator spec, right ? If that's so a constant shared by the relevant pieces of code would be less error prone I guess.